src/idd/idd.base.js

```diff
@@ -280,6 +282,8 @@ var _initializeInteractiveDataDisplay = function () { // determines settings dep
         // The flag is reset when any other update request is received.
         var _suppressNotifyBoundPlots = false;
         var _tooltipSettings = undefined;
+        var _verticalBoundaryLines = [];
```

Comment:
Why other plots do not stored here while boundary lines are stored?
Why are they different?

That makes base plot to depend on one of its derived classes which seems odd.

Reply:
Removed these lines, my mistake. Got left from my previous version of implementation of the functionality.